packages/api/.eslintignore

```diff
@@ -3,3 +3,4 @@ dist/
 readabilityjs/
 src/generated/
 test/resolvers/
+apollo.config.js
```

Comment:
For Apollo extension to work in VSCode

Reply:
nice. It only works in dev mode, right?

Reply:
yeah, but actually i don't really need to add this line because `apollo.config.js` is a Javascript file